crates/storage/src/collections/stash/mod.rs

```diff
@@ -33,6 +33,9 @@ use crate::{
 };
 use ink_primitives::Key;
 
+#[cfg(doc)]
+use crate::traits::SpreadLayout;
+
```

Comment:
Why is this needed in `cfg(doc)`?

Reply:
Needed for the crate documentation of `storage`:
```
ink/crates/storage
cargo doc --document-private-items --no-deps
```

Thing is that it's only needed for `--document-private-items`, so not sure which policy is best suited for that. IMHO we should ensure that all doc links are working, no matter if private or public. On the other hand we only test the public interface for the crates in CI currently. Wdyt?